buggy_utils.py

```
@@ -0,0 +1,107 @@
+
+
+# Mutable default argument
+def append_to_list(item, lst=[]):
+    lst.append(item)
+    return lst
+
+
+# Insecure deserialization
+import pickle
+def load_user_data(data):
+    return pickle.loads(data)  # arbitrary code execution
```

Ruff (0.15.12)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)